kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.testing.collection;
+import java.util.*;
+import java.lang.*;
+
+public class OddNumbersExterminator {
+    ArrayList<Integer> even;
+
+
+    public OddNumbersExterminator() {
+        even = new ArrayList<Integer>();
+    }
+
+    public void exterminate(ArrayList<Integer> number) {
+
+        ArrayList<Integer> even = new ArrayList<Integer>();
+        int counter = number.size();
+        if (counter == 0) {
+            System.out.println("List empty: " + counter);
```

Comment:
ta petla for ponizej poradzi sobie z pusta lista (bedzie po prostu 0 iteracji). Ten if jest zbedny